react-components/src/talons/Cart/useAddToCartEvent.js

```diff
@@ -41,10 +41,23 @@ const useAddToCartEvent = (props = {}) => {
 
     useEventListener(document, 'aem.cif.add-to-cart', async event => {
         const items = typeof event.detail === 'string' ? JSON.parse(event.detail) : event.detail;
-        const physicalCartItems = items.filter(item => !item.virtual).map(productMapper);
-        const virtualCartItems = items.filter(item => item.virtual).map(productMapper);
-        const bundleCartItems = items.filter(item => item.bundle).map(bundledProductMapper);
-        const giftCardCartItems = items.filter(item => item.giftCard).map(giftCardProductMapper);
+        const useUidItems = items
```

Comment:
trivial but it would be good to have a comment here explaining what the difference between the items with `useUid` is and why we keep the "old way of things".